dblookupext/historyRepository.go

```diff
@@ -265,7 +291,29 @@ func (hr *historyRepository) GetMiniblockMetadataByTxHash(hash []byte) (*Miniblo
 		return nil, err
 	}
 
-	return hr.getMiniblockMetadataByMiniblockHash(miniblockHash)
+	hashSize := hr.hasher.Size()
+	if len(miniblockHash) == hashSize {
+		return hr.getMiniblockMetadataByMiniblockHash(miniblockHash)
+	}
+
+	prefixedHashSize := hashSize + 1
+	if len(miniblockHash) != prefixedHashSize {
+		return nil, ErrInvalidHashSize
+	}
+
+	prefix := miniblockHash[0]
+	actualHash := miniblockHash[1:]
```

Comment:
The actualHash is the relayTxHash, and it appears you are not returning this
It might be needed to create the link to the relayTx that included this inner tx